cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsCloudNioFileProvider.scala

```diff
@@ -33,32 +30,23 @@ class DrsCloudNioFileProvider(scheme: String,
     }
   }
 
-
-  override def existsPath(cloudHost: String, cloudPath: String): Boolean =
-    checkIfPathExistsThroughMartha(getDrsPath(cloudHost, cloudPath)).unsafeRunSync()
-
+  override def existsPath(drsPath: String, unused: String): Boolean =
+    checkIfPathExistsThroughMartha(drsPath).unsafeRunSync()
```

Comment:
TOL: If I had a vote on the next piece of tech debt it'd be really nice to tidy up...